src/NuGet.Status/Web.config

```diff
@@ -64,7 +64,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30AD4FE6B2A6AEED" culture="neutral"/>
-        <bindingRedirect oldVersion="0.0.0.0-11.0.0.0" newVersion="11.0.0.0"/>
+        <bindingRedirect oldVersion="0.0.0.0-13.0.1" newVersion="13.0.1"/>
```

Comment:
Shouldn't it be `13.0.0.0` here (see yesterday [change](https://github.com/NuGet/NuGet.Licenses/blob/5529328b03f5488600c23e60e7614f1b1521fac7/src/NuGet.Licenses/Web.config#L227))? Does it start as is?

Reply:
Yep. assembly version is `13.0.0.0`  https://nuget.info/packages/Newtonsoft.Json/13.0.1